src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Mapping/RazorLanguageQueryEndpoint.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
 using Microsoft.CodeAnalysis.Razor.DocumentMapping;
 using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
```

Comment:
Is this needed or unnecessary?

Reply:
Yes, it's the namespace that currently contains IDocumentSnapshotExtensions which contains GetGeneratedOutputAsync() which I made an extension method per @davidwengier suggestion.